xchange-livecoin/src/main/java/org/knowm/xchange/livecoin/LivecoinAdapters.java

```diff
@@ -235,7 +243,7 @@ public static UserTrade adaptUserTrade(Map map) {
         price,
         DateUtils.fromMillisUtc(Long.valueOf(map.get("date").toString())),
         id,
-        map.get("externalKey").toString(),
+        Optional.ofNullable(map.get("externalKey")).map(Object::toString).orElse(null),
```

Comment:
in the documentation I see a "external" property, maybe in production its different
either way it can be nulled